CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ set(ANDROID_MKE2FS_NAME "mke2fs.android")
 # Version of android-tools and the version of boringssl being used.
 # See: https://android.googlesource.com/platform/external/boringssl/+/platform-tools-${ANDROID_VERSION}/BORINGSSL_REVISION
 set(ANDROID_VERSION 31.0.0)
-set(BORINGSSL_VERSION 2fb729d4f36beaf263ad85e24a790b571652679c)
+set(BORINGSSL_VERSION c1e156ae16fa8b61af9b5d2b74e59d3f86e141be)
```

Comment:
it makes sense to squash this change into the original 31.0.0 version bump commit